src/site/content/en/blog/min-max-clamp/index.md

```diff
@@ -97,7 +97,7 @@ conjunction with `calc()` (i.e. `font-size: max(calc(0.5vw - 1em), 2rem)`),  or
 composed (i.e. `font-size: max(min(0.5vw, 1em), 2rem)`). 
 
 {% Aside %} When using a calculation inside of a `min()`, `max()`, or `clamp()`
-function, you can nix the `calc()`. For example, writing `font-size:
+function, you can mix the `calc()`. For example, writing `font-size:
```

Comment:
Actually, I think 'nix' is the right word here. For this to make sense, the formulas in the following line need to be reversed.

@una, please weigh in.

Reply:
nix is correct

Reply:
You can remove the calc() might be more clear?

Reply:
"Nix" is technically correct but this PR is evidence that using a less common word causes confusion. So, yes, let's replace.

Reply:
```suggestion
function, you can remove the call to `calc()`. For example, writing `font-size:
```